core/amber/src/main/scala/edu/uci/ics/texera/web/ComputingUnitMaster.scala

```diff
@@ -183,6 +183,7 @@ class ComputingUnitMaster extends io.dropwizard.Application[Configuration] with
           // rely on the server-side result cleanup logic.
           case OpResultStorage.MONGODB =>
             MongoDatabaseManager.dropCollection(collectionName)
+          case OpResultStorage.ICEBERG =>
```

Comment:
Add some comments to explain why it's blank.

Reply:
I merged it with the MEMORY.  Its the same reason as the MEMORY case.